### ISSUE TITLE
Accessibility: `scratchr2/forums.css` → `.blockpost .box-head a` is not accessible

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#6403]
`.blockpost .box-head a` has a style rule that sets `color` to `var(--darkWww-page-transparentText, #b3b3b3)`, which does not have great contrast when you need it on https://scratch.mit.edu/discuss/search/?action=show_user&show_as=posts.

![List when clicking "Show your posts"](https://github.com/ScratchAddons/ScratchAddons/assets/106490990/280e3524-7680-4a99-900b-b29ab1b5135a)
